MAUI/Scheduler/appointments.md

```diff
@@ -467,6 +467,30 @@ this.Scheduler.AppointmentsSource = Meetings;
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/RecurringAppointment/BusinessObject)
 
+### Recurence appointment on last day of month
```

Comment:
Mention -1 scenario in table - https://help.syncfusion.com/maui/scheduler/appointments#recurrence-appointment

![image](https://user-images.githubusercontent.com/92789666/175527365-9fa692e7-5e56-47be-a5d3-614c692a699d.png)